src/file.cpp

```diff
@@ -484,12 +484,12 @@ void SyncFileGet::prepare()
                 transfer->localfilename = sync->localdebris;
                 sync->client->fsaccess->mkdirlocal(transfer->localfilename, true);
 
-                transfer->localfilename.appendWithSeparator(tmpname, true, sync->client->fsaccess->localseparator);
+                transfer->localfilename.appendWithSeparator(tmpname, true, LocalPath::localPathSeparator);
```

Comment:
Hi Javier, since this is constant now, we don't need to pass it at all - we can simplify these functions and the calls by removing the parameter, and just use the constant in the implementation.   thanks!

Reply:
Yeah, first I refactored automatically and then I were going to remove not necessary parameters in methods.